cmd/istioctl/main.go

```diff
@@ -271,7 +300,7 @@ func init() {
 	rootCmd.AddCommand(putCmd)
 	rootCmd.AddCommand(getCmd)
 	rootCmd.AddCommand(deleteCmd)
-	rootCmd.AddCommand(injectCmd)
+	// rootCmd.AddCommand(injectCmd)
```

Comment:
Let's keep this in. I'd suggest moving kube inject variables to inject.go to make it easy to track dependencies.